web/src/constants/dimensions.ts

```diff
@@ -16,8 +16,8 @@ export type DimensionsType = {
 const dimensions: DimensionsType = {
   maxWidthViewportSmall: 768,
   smallViewport: '(max-width: 768px)',
-  mediumViewport: '(min-width: 768px) and (max-width: 1100px)',
-  largeViewport: '(min-width: 768px)',
+  mediumViewport: '(min-width: 769px) and (max-width: 1100px)',
+  mediumLargeViewport: '(min-width: 769px)',
   minMaxWidth: 'screen and (min-width: 1100px)',
   maxWidth: 1100,
```

Comment:
Could you adjust those two values as well please to `1101`?

Reply:
could you help me why we would change it to 1101?

Reply:
Because otherwise for a `1100` px width both medium and large screen media queries match which leads to undefined behavior